tcms/core/middleware.py

```diff
@@ -32,3 +34,23 @@ def process_request(self, request):
                       }
                 )
             )
+
+
+class CheckUnappliedMigrationsMiddleware(MiddlewareMixin):
+    def process_request(self, request):
+        doc_url = ('https://kiwitcms.readthedocs.io/en/latest/installing_'
+                   'docker.html#initial-configuration-of-running-container')
```

Comment:
If you use the following syntax:

```
"line 1"
"line 2"
```

both strings will be concatenated together and there is no need to use brackets. Maybe that works with single quotes as well, IDK - I've never tried it.

Also try splitting the URL along its natural dividers like `/`

Reply:
do you mean like:
`doc_url = "https://kiwitcms.readthedocs.io/en/latest/"`
`"installing_docker.html#initial-configuration-of-running-container"`

As I'm getting a pylint error `String statement has no effect pylint(pointless-string-statement)` Is there an example I can refer to?

Reply:
ohh, never mind, I missed the \